docs/examples/en/loaders/GLTFLoader.html

```diff
@@ -182,6 +182,18 @@ <h3>[method:null load]( [param:String url], [param:Function onLoad], [param:Func
 		Begin loading from url and call the callback function with the parsed response content.
 		</p>
 
+		<h3>[method:Promise loadAsync]( [param:String url], [param:Function onProgress] )</h3>
```

Comment:
Shouldn't this method be documented here instead?

https://threejs.org/docs/index.html#api/en/loaders/Loader

Reply:
> Shouldn't this method be documented here instead?

I added the GLTFLoader docs as an example. My intention is to add this to all the loader docs, including Loader. 

Although we usually only document methods on the base class, in this case, I'm not sure that's enough - I don't think many people check the docs for the Loader class. I'd rather document it where people will discover it even if that means some duplication.

Reply:
> What do you think?

TBH, I would not start with this. Otherwise it will be copied for other cases and we end up with a lot of duplicated documentation. 

IMO even if it's not so visible, the method should be only documented in `Loader`.

Reply:
OK, well that will make my job easier in any case 😅